src/core/Dropdown/index.tsx

```diff
@@ -57,7 +67,7 @@ export interface DropdownProps<Multiple> {
 }
 
 // eslint-disable-next-line sonarjs/cognitive-complexity
-const Dropdown = <Multiple extends boolean | undefined = false>({
+const Dropdown = <Multiple extends boolean | undefined = undefined>({
```

Comment:
hmm line 73 is `multiple = false`, so I'm thinking the type needs to be `Multiple extends boolean | undefined = false` too? Unless I missed something 😆 

Thank you!

Reply:
Aah, that makes sense. I'll fix it. 🥸